tests/unit/src/main/scala/bill/Bill.scala

```diff
@@ -177,17 +176,18 @@ object Bill {
         params: DependencySourcesParams
     ): CompletableFuture[DependencySourcesResult] = {
       CompletableFuture.completedFuture {
-        val sources = CoursierSmall.fetch(
-          new Settings().withDependencies(
-            List(
-              new Dependency(
-                "org.scala-lang",
-                "scala-library",
-                mtags.BuildInfo.scalaCompilerVersion
-              )
+        val sources = Fetch
+          .create()
+          .withDependencies(
+            Dependency.of(
```

Comment:
can the dependency be extracted to a variable? Would make the invocation chain more readable